action/sealedenvelope.go

```diff
@@ -69,6 +93,13 @@ func (sealed *SealedEnvelope) Hash() (hash.Hash256, error) {
 // SrcPubkey returns the source public key
 func (sealed *SealedEnvelope) SrcPubkey() crypto.PublicKey { return sealed.srcPubkey }
 
+func (sealed *SealedEnvelope) Address() address.Address {
```

Comment:
need a comment like // Address() to satisfy `go lint`
you run `make lint` local to make sure no lint error

Reply:
change to `SenderAddress`, it's a better/clear name